src/helpers/shared/user-login.ts

```diff
@@ -24,6 +24,7 @@ import { stringify } from 'qs';
 import { loginUser } from './active-user';
 // @ts-ignore
 import Cookies from 'js-cookie';
+import { isStandalone } from './helpers';
```

Comment:
Might not need this, I don't see `isStandalone` being used anywhere in this file

Reply:
Agree - will remove it.